unittests/test_schedulers.py

```diff
@@ -632,6 +649,16 @@ def test_sched_access_constraint_partition(self):
         self.prepare_job()
         self.assertEqual(self.testjob.num_tasks, 4)
 
+    def test_sched_access_partition(self):
+        self.testjob._sched_access = ['--partition=p1']
+        self.prepare_job()
+        self.assertEqual(self.testjob.num_tasks, 16)
```

Comment:
If you the flex alloc policy is `idle`, I think that this should be 8.

Reply:
If not defined explicitly, `_sched_flex_alloc_tasks` is set to `all` already in `setUp`